disnake/audit_logs.py

```diff
@@ -532,8 +532,8 @@ class AuditLogEntry(Hashable):
     ----------
     action: :class:`AuditLogAction`
         The action that was done.
-    user: :class:`abc.User`
-        The user who initiated this action. Usually a :class:`Member`\\, unless gone
+    user: Union[:class:`Member`, :class:`User`, :class:`Object`]
+        The user who initiated this action. Usually :class:`Member`\\, unless gone
         then it's a :class:`User`.
```

Comment:
would be nice a `.. versionadded::` mentioning the change

Reply:
2.8?

Reply:
A `.. versionchanged:: 2.8` with a short description would be great, yea